### ISSUE TITLE
Send summary email of all docking evaluations to internal celpp group

### ISSUE THREAD

**Issue by coleslaw481** [drugdata/D3R#100]
Create reporting tool postevaluation.py that generates a summary of all the docking evaluations that have run. This tool will be called by the new post evaluation stage issue #111. 

This summary should basically be a table that has the Average row from RMSD.txt for each docking evaluation.  Here is an example:

```Bash
               Target_PDBID        LMCSS     SMCSS     hiResApo  hiResHolo hiTanimoto 
XX             Average             7.248     5.589     8.871     5.333     6.402     
YY             Average             6.228     6.769     7.262     6.357     5.893     
12345(XXXX)    Average             7.141     6.286     6.331     5.849     7.809     
12345_2(XXXX)  Average             4.997     4.756     6.064     4.837     5.393     
12345(XXXXX)   Average             7.162     7.193     9.448     6.511     8.192     
12345_2(XXXXX) Average             6.416     6.318     7.174     6.391     6.730     
```
See new table idea below

**Comment by coleslaw481** [comment]
Shuai added post_evaluation.py script Closing this ticket assuming that script works. Still need to write post evaluation stage, but will leave ticket issue #111 to track progress on that

**Comment by coleslaw481** [comment]
New plan Do this:

```Bash
hiResApo submissionID  #Docked #Dockable Average
------------------------------------------------
33567_autodockvina     12      13        5.73
33567_glide            11      13        5.54


hiResHolo submissionID  #Docked #Dockable Average
-------------------------------------------------
33567_autodockvina      24      27        4.26
33567_glide             22      27        4.04
```

**Comment by coleslaw481** [comment]
Fixed in 6e81574..f02185b  chrisdev -> chrisdev
